src/context/WalletContext.tsx

```diff
@@ -83,7 +83,7 @@ const WalletProvider = ({ children }: WalletProviderProps) => {
 
         const network = await provider.getNetwork()
 
-        if (network.chainId !== parseInt(process.env.ETH_CHAIN_ID || "3")) {
+        if (network.chainId !== parseInt(process.env.NEXT_PUBLIC_ETH_CHAIN_ID || "3")) {
```

Comment:
Now since we are using env files so we need NEXT_PUBLIC_ to get these variable values at frontend. If we do not add this prefix in the variable name we won't be able to get the values. It will return undefined